<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>labelswindow</class>
 <widget class="QDialog" name="labelswindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Labels Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="addLabelButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>&amp;Add Label</string>
       </property>
       <property name="icon">
        <iconset resource="frida.qrc">
         <normaloff>:/icons/document-edit.svg</normaloff>:/icons/document-edit.svg</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+A</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="importButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>&amp;Import Global Labels</string>
       </property>
       <property name="icon">
        <iconset resource="frida.qrc">
         <normaloff>:/icons/document-import.svg</normaloff>:/icons/document-import.svg</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+I</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="exportButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>&amp;Export Global Labels</string>
       </property>
       <property name="icon">
        <iconset resource="frida.qrc">
         <normaloff>:/icons/document-export.svg</normaloff>:/icons/document-export.svg</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+E</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="doneButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>&amp;Done</string>
       </property>
       <property name="icon">
        <iconset resource="frida.qrc">
         <normaloff>:/icons/document-close.svg</normaloff>:/icons/document-close.svg</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelLocalLabels">
         <property name="text">
          <string>Local Labels</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableLocalLabels">
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>21</number>
         </attribute>
         <column>
          <property name="text">
           <string>Address</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Label</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelGlobalLabels">
         <property name="text">
          <string>Global Labels</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableGlobalLabels">
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>21</number>
         </attribute>
         <column>
          <property name="text">
           <string>Address</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Label</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionChange_To_Local_Label">
   <property name="text">
    <string>Change To Local Label</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
  <action name="actionChange_To_Global_Label">
   <property name="text">
    <string>Change To Global Label</string>
   </property>
   <property name="toolTip">
    <string>Change To Global Label</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </action>
  <action name="actionDelete_Label">
   <property name="text">
    <string>Delete Label</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="frida.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionChange_To_Local_Label</sender>
   <signal>triggered()</signal>
   <receiver>labelswindow</receiver>
   <slot>actionChange_To_Local_Label()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionChange_To_Global_Label</sender>
   <signal>triggered()</signal>
   <receiver>labelswindow</receiver>
   <slot>actionChange_To_Global_Label()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDelete_Label</sender>
   <signal>triggered()</signal>
   <receiver>labelswindow</receiver>
   <slot>actionDelete_Label()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>actionChange_To_Local_Label()</slot>
  <slot>actionChange_To_Global_Label()</slot>
  <slot>actionDelete_Label()</slot>
 </slots>
</ui>
